hawc/templates/admin/base_site.html

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block branding %}
-    <h1 id="site-name"><a href="{% url 'admin:index' %}">HAWC Administration</a></h1>
```

Comment:
reverted this change too since it's on the admin page, not our main site, so it's a different style guide